src/containers/Page.jsx

```diff
@@ -144,6 +148,7 @@ class Page extends Component {
         </div>
         <main className='mainPage mainPage--sidebarOpen'>
           {this.isASocietyPage() ? <SocietyBanner society={societyInfo.info} /> : null}
+          {updating && <div className='overlay'><UpdateLoader /></div>}
```

Comment:
Since this `overlay` class works with the `UpdateLoader`, I suggest you have it inside the component and have a separate `css` file that has the styling for the `overlay and span`. Also,  I think you should rename the `UpdateLoader` component as it could be reused elsewhere. For instance, replacing the `Loading ...` text that we use.

Reply:
The reason it is outside the `UpdateLoader` is because the `loader` doesn't necessarily require it to work. The overlay just helps it stand out.
This was done purposefully because the update loader shall be used in other parts of the system when updates are being fetched. These areas may not necessarily require the overlay when displaying the loader. They shall possibly only require resizing of the loader
That is also the reason the loader is named that way. Because this particular loader shall be used for **update requests only**.
We shall have other loaders for **GET** and **POST** requests.

I hope that helps 😉